plugins/formal-verification/formver.core/src/org/jetbrains/kotlin/formver/conversion/InlineMethodConverter.kt

```diff
@@ -20,25 +21,26 @@ sealed interface SubstitutionItem {
 
 data class SubstitutionName(override val name: MangledName) : SubstitutionItem
 
-data class SubstitutionLambda(val body: FirBlock, val args: List<Name>) : SubstitutionItem {
+data class SubstitutionLambda(val body: FirBlock, val args: List<Name>, val scopedNames: Map<Name, Int>) : SubstitutionItem {
     override val name = null
     override fun lambdaBody(): FirBlock = body
     override fun lambdaArgs(): List<Name> = args
 }
 
 class InlineNameMangler(
```

Comment:
Would it be possible to switch to having a single mangler?

Reply:
I think that we can do it